missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/misc/ScoreboardManager.java

```diff
@@ -228,11 +228,16 @@ public void resetScoreboard() {
      */
     private String replaceScoreboardPlaceholders(String text) {
 
-        String time;
+        String time = "";
         if (game.getState() == GameState.LOBBY) {
+            // Show the planned duration of the next game:
             time = Integer.toString(game.getArena().getGameDuration());
-        } else {
-            time = Integer.toString(game.getTimer().getSeconds() / 60);
+        } else if (game.getState() == GameState.INGAME) {
+            // Show the remaining duration of the running game:
+            time = Integer.toString(game.getTaskManager().getTimer().getSeconds() / 60);
+        } else if (game.getState() == GameState.END) {
+            // Show the remaining duration of the last game:
+            time = Integer.toString(game.getRemainingGameDuration() / 60);
```

Comment:
Fixes a NullPointerException when the scoreboard is updated, since now the INGAME timer is no longer queried for the `%time%` placeholder in the END phase.

Reply:
:+1: